feat: add allowedMentions to faq (#640)
discordjs/guide: guide/popular-topics/faq.md

Mentions in embeds may resolve correctly in embed description and field values but will never notify the user. Other areas do not support mentions at all.
:::

### How do I control which users and/or roles are mentioned in a message?

> Controlling which mentions (and replies) will send a ping is done via the `allowedMentions` option, which replaces `disableMentions`.

Reviewer: ```suggestion
Controlling which mentions will send a ping is done via the `allowedMentions` option, which replaces `disableMentions`.
```
Since the `allowedMentions` options works with `channel.send()` but not `message.reply()` in v12, the "(and replies)" bit should be removed for now.

This can be set as a default in `ClientOptions`, and controlled per-message sent by your bot.
```diff

> - new Discord.Client({ disableMentions: "everyone" });
> + new Discord.Client({ allowedMentions: { parse: ["users", "roles"], repliedUser: true });

Reviewer: ```suggestion
- new Discord.Client({ disableMentions: 'everyone' });
+ new Discord.Client({ allowedMentions: { parse: ['users', 'roles'] });
```
`repliedUser` seems to be a v13 option. The paragraph above may need editing.

```

> Even more control can be achieved by listing specific `users` or `roles` to be mentioned by ID, eg:

Reviewer: ```suggestion
Even more control can be achieved by listing specific `users` or `roles` to be mentioned by ID, e.g.:
```

```js

> message.channel.send('<@830569888157925434> <@122157285790187530>', {
> 	allowedMentions: { users: ['830569888157925434'] }

Reviewer: ```suggestion
message.channel.send('<@123456789012345678> <@987654321098765432>', {
	allowedMentions: { users: ['123456789012345678'] },
```
These user IDs should use placeholder values
